### DISCUSSION TITLE
Copy model/field comments into generated schema typings

### DISCUSSION THREAD

**Discussion by CDDelta** [prisma/prisma#4912] [Q&A] [answered]
It would be great if the comments I have defined on fields and models in my `schema.prisma` file are copied into the generated TypeScript typings.

It will help a lot to have said comments available during development!

**Comment by ryands17** [comment, marked as answer]
Hey @CDDelta 👋 

This seems like a feature request so I would suggest creating the same [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with your use case so that we can look into this :)

**Reply by CDDelta** [reply]
Done here #5113 😄

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
